views/js/controller/Delivery/monitoring.js

```diff
@@ -40,28 +42,45 @@ define([
      * The CSS scope
      * @type {String}
      */
-    var cssScope = '.delivery-manager';
+    var cssScope = '.delivery-monitoring';
 
     // the page is always loading data when starting
     loadingBar.start();
 
     /**
-     * Controls the ProctorDelivery index page
+     * Formats a time value to string
+     * @param {Number} time
+     * @returns {String}
+     * @private
+     */
+    var _timerFormat = function(time) {
+        return __('%d min', Math.floor(time / 60));
+    };
+
+    /**
+     * Controls the taoProctoring delivery page
      *
-     * @type {{start: Function}}
+     * @type {Object}
      */
     var proctorDeliveryIndexCtlr = {
         /**
          * Entry point of the page
          */
         start : function start() {
-            var $list = $(cssScope + ' .list');
-            var dataset = $list.data('set');
-            var deliveryId = $list.data('id');
-            var assignUrl = helpers._url('testTakers', 'ProctorDelivery', 'taoProctoring', {id : deliveryId});
-            var removeUrl = helpers._url('remove', 'ProctorDelivery', 'taoProctoring', {id : deliveryId});
-            var authoriseUrl = helpers._url('authorise', 'ProctorDelivery', 'taoProctoring', {id : deliveryId});
-            var serviceUrl = helpers._url('deliveryTestTakers', 'ProctorDelivery', 'taoProctoring', {id : deliveryId});
+            var $container = $(cssScope);
+            var $list = $container.find('.list');
+            var crumbs = $container.data('breadcrumbs');
+            var dataset = $container.data('set');
+            var deliveryId = $container.data('delivery');
+            var testCenterId = $container.data('testcenter');
+            var assignUrl = helpers._url('testTakers', 'Delivery', 'taoProctoring', {delivery : deliveryId, testCenter : testCenterId});
+            var removeUrl = helpers._url('remove', 'Delivery', 'taoProctoring', {delivery : deliveryId, testCenter : testCenterId});
+            var authoriseUrl = helpers._url('authorise', 'Delivery', 'taoProctoring', {delivery : deliveryId, testCenter : testCenterId});
+            var serviceUrl = helpers._url('deliveryTestTakers', 'Delivery', 'taoProctoring', {delivery : deliveryId, testCenter : testCenterId});
+
+            var bc = breadcrumbsFactory($container, crumbs);
+
+            //@TODO format the incoming data before displaying in the datatable
```

Comment:
agree formatting the data before would be more efficient than using the datatable transform  on each colums